announcements/2020-Week-30/aws.md

```diff
@@ -0,0 +1,13 @@
+# :zap: This Week in AWS Releases (July 24, 2020):zap:
+
+[AWS v9.3.5](https://github.com/giantswarm/releases/tree/master/aws/v9.3.5) updates NGINX Ingress Controller to the latest release and fixes a regression introduced in the previous upstream version.
```

Comment:
We included also upgrade to k8s 1.16.12 (see https://github.com/giantswarm/releases/tree/master/aws/v9.3.5#kubernetes-11612)

Should it be covered here? //cc @MartaD

Reply:
Yes, thank you, it’s there now.